hw/ip/tlul/dv/cov/xbar_cov_excl.el

```diff
@@ -0,0 +1,877 @@
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: weicai
+// Format Version: 2
+// Date: Tue Nov  5 16:03:15 2019
+// ExclMode: default
+//==================================================
+CHECKSUM: "4068824375 1452893182"
+MODULE: prim_arbiter ( parameter N=3,DW=102 ) 
+ANNOTATION: "[UNR]"
+Condition 2 "1009385265" "(arb_valid && ((!arb_ready))) 1 -1" (2 "10")
+CHECKSUM: "4068824375 2250367794"
+MODULE: prim_arbiter ( parameter N=2,DW=102 ) 
+ANNOTATION: "[UNR]"
+Condition 2 "1009385265" "(arb_valid && ((!arb_ready))) 1 -1" (2 "10")
+CHECKSUM: "100122082 3741747543"
+MODULE: tlul_assert
+ANNOTATION: "[NON_RTL]"
+Block 1 "0" "assign disable_sva = (tlul_assert_ctrl === 1'b0);"
+CHECKSUM: "3373532301 1341275977"
+MODULE: prim_fifo_sync
+ANNOTATION: "[UNSUPPORTED]"
+Branch 1 "2323268504" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "[UNSUPPORTED]"
+Branch 2 "3736627057" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+CHECKSUM: "3344740064 2208587804"
+INSTANCE: tb.dut
+ANNOTATION: "[UNSUPPORTED]"
```

Comment:
Should these be [UNR] since it is impossible to hit these?

Reply:
these toggles are more like unsupported like we can drive param/user with some value but we don't support it.

Reply:
update output addresses to UNR